### PR TITLE
Added submariner operator to catalog

### PR DESCRIPTION
**Pull request by james-cccc**
Added required manifests for the submariner operator

### CODE REVIEW

File: submariner-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The current *overlays* available are for the following channels:
+* [stable-0-12](overlays/stable-0-12)
+
+## Usage
+
+If you have cloned the `gitops-catalog` repository, you can install the OpenShift Logging operator based on the overlay of your choice by running from the root `gitops-catalog` directory
+
+```bash
+oc apply -k openshift-logging/operator/overlays/<channel>
```
Comment (pittar) [inline submariner-operator/README.md:17]: Please update this to be `oc apply -k submariner-operator/overlays/<channel>`.
Thanks!

**Comment by pittar** [comment]
@james-cccc , one minor issue with the README.  Once that's fixed, I'll merge this PR.
Thanks!

File: submariner-operator/README.md (hunk 1 above)
Comment (james-cccc) [reply submariner-operator/README.md]: Good spot, thanks

**Comment by james-cccc** [comment]
@pittar - fixed the typo issue so we should be good now :)

**Comment by pittar** [comment]
Thanks for the contribution!

**Pull request merged**